packages/graphile-build-pg/src/plugins/PgViewPlugin.js

```diff
@@ -0,0 +1,132 @@
+"use strict";
+
+Object.defineProperty(exports, "__esModule", { value: true });
+
+var _queryFromResolveData = require("../queryFromResolveData");
+
+var _queryFromResolveData2 = _interopRequireDefault(_queryFromResolveData);
+
+var _debug = require("debug");
+
+var _debug2 = _interopRequireDefault(_debug);
+
+var _addStartEndCursor = require("./addStartEndCursor");
+
+var _addStartEndCursor2 = _interopRequireDefault(_addStartEndCursor);
+
+function _interopRequireDefault(obj) {
+  return obj && obj.__esModule ? obj : { default: obj };
+}
```

Comment:
You seem to have added the compiled version rather than the raw source?

Reply:
oh yea whoops. I'll fix that. I had been editing the raw source when developing it then just copied it over.